src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
```

Comment:
Use `const` for the daily rate and discount values to prevent them from being unintentionally modified later in the code. Refer to checklist item 3.